changelog/30592.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:feature
+**Custom login settings**: Adding new components to be able to view and delete custom login rules
```

Comment:
I've gone back and forth on this - this work is part of the larger custom login work so I'm not sure it needs a separate changelog (it can be rolled into the general feature changelog [here](https://github.com/hashicorp/vault/pull/30700/files)). But if you disagree, then perhaps we can reword it to cater more to the end user
```suggestion
**Custom login settings**: Adding view to list and delete custom login rules
```